bin/akula.rs

```diff
@@ -676,6 +679,7 @@ fn main() -> anyhow::Result<()> {
 
                 info!("Running staged sync");
                 staged_sync.run(&db).await?;
+                mining_stage.run(&db).await?;
```

Comment:
Will staged_sync and mining_stage be executed in sequential? 
use `tokio::spawn` to create a new thread to run mining_stage?